SQL/updates/13-14.sql

```diff
@@ -0,0 +1,35 @@
+#Updating the SQL from version 13 to version 14. -Kyet
+
+
+# Add new tables used by backend tooling
+
+CREATE TABLE `ip2group`
+(
+  `ip` varchar (18) NOT NULL,
+  `date` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP NOT NULL,
+  `groupstr` varchar (32) NOT NULL DEFAULT '',
+  PRIMARY KEY (`ip`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+ALTER TABLE `ip2group` ADD INDEX(`groupstr`);
```

Comment:
Questioning the worth of putting a closed-source backend table into the public schema if we are the only ones using it, and it wont have any ingame use, unless you plan to add some.